testplan/testing/pyunit.py

```diff
@@ -36,42 +36,56 @@ class PyUnit(testing.Test):
 
     CONFIG = PyUnitConfig
 
-    def __init__(self, name, suite, description=None, **options):
-        options.update(self.filter_locals(locals()))
-        super(PyUnit, self).__init__(**options)
-        self._suite_report = report_testing.TestGroupReport(
-            name=self.cfg.name, category="testsuite"
-        )
-
     def main_batch_steps(self):
         """Specify the test steps: run the tests, then log the results."""
         self._add_step(self.run_tests)
         self._add_step(self.log_test_results)
 
     def run_tests(self):
         """Run PyUnit and wait for it to terminate."""
+        for pyunit_suite in self.cfg.suites:
```

Comment:
when we have multiple suites, all of them will use self.cfg.name as the suite name, I think that would be a problem.

Reply:
Good spot! I think maybe this needs some rethink with adding multiple suites.